docusaurus/docs/recipes/splash-screen.md

```diff
@@ -20,14 +20,7 @@ npm i @moxy/react-wait-for-react
 
 ### 2. Add `<SplashScreen />` to your project
 
-Copy the [`splash-screen`](https://github.com/moxystudio/next-with-moxy/tree/master/docusaurus/static/recipes-assets/splash-screen) component folder into `www/shared/components` and edit `www/shared/components/index.js` to export it:
```

Comment:
I think we need to rename the splash-screen folder to `react-splash-screen`, is that right?